spinn_front_end_common/interface/abstract_spinnaker_base.py

```diff
@@ -492,6 +495,7 @@ def __init__(
         self._no_machine_time_steps = None
         self._minimum_auto_time_steps = self._config.getint(
                 "Buffers", "minimum_auto_time_steps")
+        self._run_until_complete = False
```

Comment:
id personally move that above the things where we start reading configs. but your welcome to ignore this one